src/Lecture_3/Homework_3/Task_13.java

```diff
@@ -0,0 +1,61 @@
+package Lecture_3.Homework_3;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Task_13 {
+
+    public static void main(String[] args) {
+
+        Scanner sn = new Scanner(System.in);
+
+        System.out.println("Please enter the number of rows to form square two dimensional array");
+        int rows = sn.nextInt();
+        System.out.println("Please enter the number of columns to form square two dimensional array");
```

Comment:
In order to have diagonals your array must be square. You need only 1 user input, not 2.

Reply:
Thanks, that makes sense. Corrected code to accept only one input, excluded validation for equal raws and columns values.